content/getting-started/accept-invitation.md

```diff
@@ -39,3 +39,14 @@ invitation page again, but this time tap the 'Open the app' button on that
 page to launch the app and begin the account setup.
 
 As part of the account setup process, you will choose a username and password.
+
+## Step 4: Understanding circles and contacts
+
+Once you've created your account and opened the Snikket app, you'll see that
+you've automatically been invited to a group chat.
```

Comment:
```suggestion
you've automatically been added to a group chat.
```

Snikket apps auto-accept invites, but the circle group functionality doesn't actually use invites anyway (the group is added directly to the user's bookmarks).

Reply:
Ah yep, fixed up @mwild1.